CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # TelegramBotAPI changelog
 
+## 0.37.4
+* `Core`:
+    * New `SupergroupEvent` subtype: `MigratedToSupergroup`. This event is sent when a group is converted to a supergroup while bot is in the group.
```

Comment:
I think, that description (`This event is sent when a group is converted to a supergroup while bot is in the group.`) should be in the kdocs instead of changelog

Reply:
I didn't know there are KDocs :) I added it to the `MigratedToSupergroup` class, but also left the line in the changlelog.